docker-compose.yml

```diff
@@ -99,6 +99,10 @@ services:
       - ISSUANCE_SCHEMA_POLICY=EXISTING_OR_NEW
       - ISSUANCE_CRED_DEF_POLICY=EXISTING_OR_NEW
       - DEMO_MODE=true
+      - SMTP_HOST=localhost
+      - SMTP_PASS=
+      - SMTP_PORT=2525
+      - SMTP_USER=
```

Comment:
Let's keep the `smtp` connection internal to the docker network if we can. Cahnge `SMTP_HOST` to `smtp4dev` and `SMTP_PORT` to `25`.